jetty-unixdomain-server/src/main/config/etc/jetty-unixdomain.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="addConnector">
+    <Arg>
+      <New id="unixDomainConnector" class="org.eclipse.jetty.unixdomain.server.UnixDomainServerConnector">
+        <Arg><Ref refid="Server" /></Arg>
+        <Arg type="int"><Property name="jetty.unixdomain.acceptors" default="1"/></Arg>
+        <Arg type="int"><Property name="jetty.unixdomain.selectors" default="-1"/></Arg>
+        <Arg>
+          <Array type="org.eclipse.jetty.server.ConnectionFactory">
+            <Item>
+              <New class="org.eclipse.jetty.server.HttpConnectionFactory">
```

Comment:
Is `HTTP` the only protocol that we will deliver over UnixDomain sockets?
Can we do `SSL`, `Proxy`, `HTTP/2` ? 

At the very least, this file should be named `jetty-unixdomain-http.xml`

Reply:
@gregw I agree, but the current status for the other implementation is that we have 7 modules for all possible permutations.

I wanted to explore the possibility to separate the "carrier", i.e. IP vs Unix-Domain, from the protocols you attach to it.
Something along the lines of `--add-modules=http2` would implicitly use IP (i.e. `ServerConnector`), while `--add-modules=unix-domain,http2` would use Unix-Domain (i.e. `UnixDomainServerConnector`).

I thought that this could be worked out in a different PR.

Reply:
Working out the full topology can be punted to a different PR, but you need to at least start with the right name.    This should be `unixdomain-http` with initially can be complete in itself, but in future might depend on a `unix-domain` module.      If you start with the non protocol module name, then we have less flexibility in future

Reply:
Done.